specs/classroom_spec.rb

```diff
@@ -0,0 +1,24 @@
+relative_require '../classroom'
```

Comment:
- [ ] the tests are not running correctly because in this line you are using `relative_require` however the keyword you should use is `require_relative`, what you can do to avoid this kind of mistake is to enable an autofill extension in VS code for example.

Reply:
Thank you so much. We'll work on making the test pass locally

Reply:
I tried reaching out to you via zoom but I wasn't able to get a hold of you, when downloading the repo now and running `rspec .` it works fine on my side, maybe changed something additional to what I suggested. check if it works if you stash the changes you made on your local machine right now and pull this branch again.

Reply:
Thank you so much @mikethreels, I think the problem has to do with my system:
![Screenshot from 2022-03-23 17-12-36](https://user-images.githubusercontent.com/52934128/159745541-815d3569-792a-40f1-b757-f188a36747ce.png)

Reply:
@mikethreels Thank you so much.